main.go

```diff
@@ -335,9 +343,20 @@ func mainerr() error {
 			case *fVersion:
 				fmt.Printf("%v %v\n", mp.Module.Path, mp.Module.Version)
 			case *fRun:
-				argv := append([]string{target}, runArgs...)
-				if err := syscall.Exec(argv[0], argv, os.Environ()); err != nil {
-					return fmt.Errorf("failed to exec %v: %v", target, err)
+				if runtime.GOOS == "windows" {
+					run := exec.Command(target, runArgs...)
+					run.Stdout = os.Stdout
+					run.Stderr = os.Stderr
+					if err := run.Run(); err != nil {
+						if _, ok := err.(*exec.ExitError); !ok {
+							return fmt.Errorf("failed to run %v: %v", strings.Join(run.Args, " "), err)
+						}
+					}
+				} else {
+					argv := append([]string{target}, runArgs...)
+					if err := syscall.Exec(argv[0], argv, os.Environ()); err != nil {
```

Comment:
Per Slack conversation, you're referring to `os/exec` above. Reasoning is similar to the answer I gave to @mvdan (can't recall the medium): I was simply looking to keep the OS process list as tidy as possible. Otherwise "everything" will be `gobin ...`. 

But having just written that, I suspect my thinking here is flawed. Because by exec-ing, the `os.Args[0]` value will be the long, complicated path to the cached binary. Which will arguably be even less clear. I'm tempted to make things consistent using `os/exec` as you suggest therefore. 

Thanks both for talking this one through.

Changes in a follow-up PR.